config/routes.rb

```diff
@@ -526,6 +526,10 @@ def patch(*)
 
     resources :reviewable_claimed_topics, only: %i[create destroy]
 
+    resources :reviewables, only: [] do
+      resources :reviewable_notes, only: %i[create destroy], path: "notes"
```

Comment:
```suggestion
      resources :reviewable_notes, only: %i[create destroy], path: "notes", constraints: StaffConstraint.new
```

Additional safeguard at the route level.